lonestar/analytics/gpu/sgd/SGDGraphCu.h

```diff
@@ -151,21 +151,21 @@ struct SGD_LC_LinearArray_Undirected_Graph {
     // node_data
     memset(node_data(), 0, sizeof(unsigned int) * _num_nodes);
     for (unsigned int i = 0; i < _num_edges; ++i) {
-      out_neighbors()[i] = *le64toh(outs + i);
```

Comment:
how did the previous version build at all?

- perhaps `le32toh(outs[i])` instead (and throughout).

Reply:
Not sure. @chenxuhao  is owning that app and was building it. He can answer better.

Reply:
are we going to keep the `*(outs +i)` form or change it to `outs[i]`?

Reply:
Yes, I am going to change it. Thanks